static/js/components/EducationForm.js

```diff
@@ -138,7 +189,9 @@ class EducationForm extends ProfileFormFields {
           <Cell col={7} />
           <Cell col={1}>
             <Switch
-              onChange={()=>{this.handleSwitchClick(level.value);}}
+              id={`profile-tab-education-switch-${level.value}`}
+              onClick={()=>{this.handleSwitchClick(level.value);}}
+              readOnly={true}
```

Comment:
I see, it looks like it's expecting us to set `onChange` instead of `onClick` here. Can you make this change here and for the employment tab?